tools/accuracy_checker/accuracy_checker/annotation_converters/yolo_labeling_converter.py

```diff
@@ -0,0 +1,86 @@
+limitations under the License.
+"""
+
+import numpy as np
+from .format_converter import BaseFormatConverter, ConverterReturn
+from ..utils import read_txt, check_file_existence
+from ..config import PathField
+from ..representation import DetectionAnnotation
+
+
+class YOLOLabelingConverter(BaseFormatConverter):
+    __provider__ = 'yolo_labeling'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            'data_dir': PathField(is_directory=True, description='Directory with annotations and images'),
+            'labels_file': PathField(description='Labels file')
+        })
+        return params
+
+    def configure(self):
+        self.data_dir = self.get_value_from_config('data_dir')
+        self.labels_file = self.get_value_from_config('labels_file')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        content_errors = None if not check_content else []
+        image_ann_pairs = []
+
+        annotation_files = self.data_dir.glob('*.txt')
+        for ann_file in annotation_files:
+            image_ann_pairs.append((ann_file.name.replace('txt', 'jpg'), ann_file.name))
+
+        num_iterations = len(image_ann_pairs)
+        annotations = []
+        for idx, (identifier, annotation_file) in enumerate(image_ann_pairs):
+            labels, x_mins, y_mins, x_maxs, y_maxs = self.parse_annotation(annotation_file)
+            annotations.append(DetectionAnnotation(identifier, labels, x_mins, y_mins, x_maxs, y_maxs))
+            if check_content:
+                check_file_existence(self.data_dir / identifier)
+                content_errors.append('{}: does not exist'.format(self.data_dir / identifier))
+            if progress_callback and idx % progress_interval == 0:
+                progress_callback(idx * 100 / num_iterations)
+
+        meta = self.generate_meta()
+
+        return ConverterReturn(annotations, meta, content_errors)
+
+    def parse_annotation(self, annotation_file):
+        labels, x_mins, y_mins, x_maxs, y_maxs = [], [], [], [], []
+        for line in read_txt(self.data_dir / annotation_file):
+            label, x, y, width, height = line.split()
+            x_min, y_min, x_max, y_max = self.convert_from_yolo_to_bbox(float(x), float(y), float(width), float(height))
+            labels.append(int(label))
+            x_mins.append(x_min)
+            y_mins.append(y_min)
+            x_maxs.append(x_max)
+            y_maxs.append(y_max)
+        return np.array(labels), np.array(x_mins), np.array(y_mins), np.array(x_maxs), np.array(y_maxs)
+
+    @staticmethod
+    def convert_from_yolo_to_bbox(x, y, width, height):
+        x_min, y_min = (x - width / 2), (y - height / 2)
+        x_max, y_max = (x + width / 2), (y + height / 2)
+        return x_min, y_min, x_max, y_max
+
+    def generate_meta(self):
+        labels = read_txt(self.labels_file)
```

Comment:
can we generate default labels (remember max label during annotation conversion and just map ranges)?

Reply:
Added